### PR TITLE
PMM-4151 SHOW INDEX action.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-376/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-376/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-376/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-376/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-376/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-376/

File: api/agentpb/agent.proto [hunk 1]
```diff
@@ -97,29 +100,35 @@ message StartActionRequest {
...
     string dsn = 1;
     string table = 2;
   }
+  // MySQLShowIndexParams describes MySQL SHOW INDEX action parameters.
+  message MySQLShowIndexParams {
+    string dsn = 1;
+    string table = 2;
+  }
   string action_id = 1;
-  management.ActionType type = 2;
```
Comment (AlekSi) [inline api/agentpb/agent.proto]: `oneof` below is enough. Additionally, it enables `go-sumtype` check.

**Pull request merged**